modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParameterValueResolver.java

```diff
@@ -20,4 +22,6 @@
    * @throws ValueResolvingException if the resolution fails
    */
   Object getParameterValue(String parameterName) throws ValueResolvingException;
+
+  Map<String, ValueResolver<? extends Object>> getParameters();
```

Comment:
maybe rename to getResolvers rather than getParameters?

Reply:
I wouldn't do that. IN this way is consistent with its sibling method and the interface name. 

Also, add jdoc and don't forget the `@since`

Reply:
done